app/src/scripts/utils/json-ld.js

```diff
@@ -0,0 +1,63 @@
+const schemaGenerator = datasets => {
+  const dataset = datasets ? datasets.dataset : null
+  const context = 'http://schema.org'
+  const type = 'Dataset'
+  let publisher = 'OpenNeuro'
+
+  if (dataset) {
+    let description = dataset.description
+    let name = description ? description.Name : null
+    let authors = formatAuthors(dataset.authors)
+    let datePublished = dataset.created
+    let dateModified = dataset.modified
+    let license = description ? description.License : null
+    let version = dataset.snapshot_version
+    let datasetDescription = dataset.README ? dataset.README : dataset.label
+
+    let schema = {
+      '@context': context,
+      '@type': type,
+      '@id': 'linktodoi',
```

Comment:
This should probably be removed if there isn't data to uniquely identify it yet.